doc/components_2.md

```diff
@@ -1,7 +1,8 @@
 ## mj-include
 
-The `mjml-core` package allows you to include external `mjml` files
-  to build your email template.
+The `mjml-core` package adds `mj-include` content to
```

Comment:
The old version is clearly better here, I don't know why would you mention "memory" nor "copy".